build.gradle

```diff
@@ -22,6 +22,7 @@ apply plugin: 'application'
 apply plugin: 'koin'
 apply plugin: "com.github.johnrengelman.shadow"
 apply plugin: 'org.jlleitschuh.gradle.ktlint'
+apply plugin: 'kotlin-kapt'
```

Comment:
@aanciaes what is this?

Reply:
This is a utility for annotation processors for kotlin, since moshi uses this for generating adapters / serializers.